src/components/molecules/RadioField/RadioField.tsx

```diff
@@ -102,16 +102,23 @@ const Input = styled.input<IInputStatus>`
   }
 `;
 
-export interface IRadioField extends IField, IInputStatus {}
+export interface IRadioField extends IField, IInput {}
 
-const RadioField: FC<IRadioField> = ({ label, hasError, errorMessage, isValid, inputProps, ...rest }) => {
-  const { value } = inputProps;
-
-  if (!inputProps.value) throw Error('Value must be set in inputProps. Check the docs.');
+const RadioField: FC<IRadioField> = ({
+  label,
+  hasError,
+  errorMessage,
+  isValid,
+  value,
+  inputSize,
+  className,
+  ...rest
```

Comment:
This is typed as a FC, in rest you will have children, is Input designed in a way to handle the children ?

Reply:
It uses [`styled.input`](https://github.com/zopaUK/react-components/blob/master/src/components/molecules/RadioField/RadioField.tsx#L55) which I think it might be fine, happy to update this here just in case 💅🏻

Reply:
Might be safer to only pass down what we need in this case, some people might find it confusing that children is accepted but not showing (or worse, break the component)

Reply:
I think if we remove `children` from the type we still can benefit from the conciseness of `...rest` 👌🏻 ( _this is an old component and so far we never had issues around this_ 💅🏻 )